### ISSUE TITLE
Use S3 Server Side Encryption in Strata

### ISSUE THREAD

**Issue by geoffbonneau** [facebookgo/rocks-strata#19]
I am not very familiar with Go, so I apologize if this is a silly question. Is there a way to have Strata upload files to S3 with the SSE flag/options enabled?

**Comment by igorcanadi** [comment]
Yeah, I think so. We use github.com/AdRoll/goamz library to access S3 and that library does expose the SSE option: https://github.com/AdRoll/goamz/blob/master/s3/s3.go#L63

So you'd just have to hook it up with our API here: https://github.com/facebookgo/rocks-strata/blob/master/strata/s3storage/storage.go#L59

Feel free to submit the pull request!

**Comment by geoffbonneau** [comment]
Thanks for the pointers, I will take a swing at it.
